addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -17,6 +17,8 @@
 _logger = logging.getLogger(__name__)
 
 
+# IAP Server address.
+# Customize this address if you want to use the IAP Test Server
 SERVER_URL = 'https://l10n-it-edi.api.odoo.com'
```

Comment:
@jco-odoo Do you think that this should also be a parameter? 
We cannot use test mode if we don't modify this line, so we would just have tests from local instances with that line  modified.

Reply:
Or a test env config param and the url hard coded?  In any case, this is not some demo environment, but an experimentation environment, which will mostly depend on people having access locally.  But no, I think your suggestion makes sense.  (but if the system param does not exist, you use this as standard)